### DISCUSSION TITLE
親指キーだけ反応しない

### DISCUSSION THREAD

**Discussion by tokuhirom** [Yowkees/keyball#447] [Q&A] [answered]
![image](https://github.com/Yowkees/keyball/assets/21084/5f52dcc2-8939-426c-8f63-ed0eed7422eb)
こちらのキーのみ反応しないです。ピンセットで、両端をショートさせたときに、このキーのみ反応しないです。
low profile の方も、通常のソケットの方も反応しない状況です。という状況なので、おそらくダイオードがおかしいのかなと思っているのですが、解決策を教えていただけるとありがたいです 🙏

**Comment by Ryo624** [comment, marked as answer]
ご連絡ありがとうございます。

写真に示しました赤色の丸のダイオードが対応するダイオードになっておりますので、キーソケットのはんだ付けと併せて見直していただけますと幸いです。
![289804633-5f52dcc2-8939-426c-8f63-ed0eed7422eb](https://github.com/Yowkees/keyball/assets/143698349/4a011400-84bd-4e1e-81f5-a66ec539c0e4)

**Reply by tokuhirom** [reply]
早速の回答ありがとうございました。色々試していたらうまく動くようになりました。
ロープロファイルのキースイッチだとうまく動かなかったのでCherryMX使うようにしたらなぜかうまく動くようになりました。。(謎)
